### PR TITLE
[9.0] document_page_environment_manual

### PR DESCRIPTION
**Pull request by naousse**
Depends on:
- [x] https://github.com/OCA/management-system/pull/95

### CODE REVIEW

File: document_page_environment_manual/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+knowledge/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+knowledge/issues/new?body=module:%20
+mgmtsystem_manuals%0Aversion:%20
```
Comment (max3903) [inline document_page_environment_manual/README.rst]: Wrong repo and module here.

File: document_page_environment_manual/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================
+Management System - Manual
+================================
```
Comment (dreispt) [inline document_page_environment_manual/README.rst]: Fix `=` to match the length of the text

File: document_page_environment_manual/README.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+To contribute to this module, please visit http://odoo-community.org.
+
+Changelog
+---------
+
+v9.0.1.0.0
+
+Here are the modification that have been done:
+
+* we renamed document_page to document_page_environment_manual in the following folder data
```
Comment (dreispt) [inline document_page_environment_manual/README.rst]: This technical detailed explanation is redundant with the git change history. Please describe only functionally relevant changes.

**Comment by dreispt** [comment]
It's OK after making CI green and fixing the README. 
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**